net/krb5/Makefile

```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=krb5
 PKG_VERSION:=1.16
-PKG_RELEASE:=1
+PKG_RELEASE:=3
```

Comment:
Did you squash something here? With onyl a single commit please bump from `1` to `2`, not `3`.

Reply:
yep was squashed, will cleanup this and the other stuff if i have some time left this week.